userService.js

```diff
@@ -0,0 +1,130 @@
+const mssql = require("mssql/msnodesqlv8");
+const dbConfig = require("./dbconfig");
+const sqlHelper = require("./userServiceSQLHelper");
+
+const pool = new mssql.ConnectionPool(dbConfig.config);
+
+function paginateUsers(req, res) {
+    pool.connect(() => {
+        var sql = "SELECT * FROM dbo.Users";
```

Comment:
i'd prefer use snakecase for tables, columns and e.t.c.

Reply:
Fixed.